content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1028,6 +1028,22 @@ last saw a request where the client authenticated using the service account toke
 If a legacy token was last used before the cluster gained the feature (added in Kubernetes v1.26),
 then the label isn't set.
 
+### kubernetes.io/legacy-token-invalid-since
+
+Type: Label
+
+Example: `kubernetes.io/legacy-token-invalid-since: 2023-10-27`
+
+Used on: Secret
+
```

Comment:
describe what the label *does* as well (makes the token contained in the Secret invalid for authentication)